### ISSUE TITLE
Tooltip Error See logs

### ISSUE THREAD

**Issue by BearsAreBetter** [mezz/JustEnoughItems#3463]
### Steps to Reproduce the Bug

--

### Expected Behavior

--

### Actual Behavior

No names for items

### Mod Pack Version (Optional)

1.20.1

**Comment by Deltanon** [comment]
Also getting this Issue; Running jei-1.20.1-forge-15.3.0.4.jar;

Rolling back to jei-1.20.1-forge-15.2.0.27.jar does fix the tooltips,
but then the client crashes when you LMB to bring up the recipe GUI.

**Comment by StrategicTrash** [comment]
I found 2 solutions for this:
1: Rolled back to jei-1.20.1-forge-15.2.0.27.jar, fix the tooltips and I don't have any crashes.
2: When I open Minecraft, If i first go into a single player map the tooltips is working and then it's working on servers as well. If I go into the server then it's not going to work, even if I go into single player, the only option is to restart.
More insight: me and a friend of mine both have the exact same mods and jar files (i sent them all the jars) and he doesn't have this problem, I do. We both play on a server that I have created and it's running on my local machine (maybe that is a problem, dunno).
